fastcrawler/core/app.py

```diff
@@ -30,39 +37,40 @@ def __init__(
         controller: ControllerProto | None = None,
     ):
         """Initialize FastCrawler with defined crawlers"""
-        ...
-        if isinstance(crawlers, Process):
-            self.crawlers = [
-                crawlers,
-            ]
-        else:
-            self.crawlers = crawlers
-
+        self.crawlers = list_process(crawlers)
         self.controller = controller or ProcessController(app=RocketryApplication())
         if not self.crawlers or len(self.crawlers) == 0:
             raise NoCrawlerFoundError
 
     @property
-    def get_all_serves(self) -> list[Callable]:
+    def get_all_serves(self) -> list[Coroutine[Any, Any, None]]:
```

Comment:
we don't use Any

Reply:
What type I should use then?
I have to look up in uvicorn to check the protocol, then define the servable coroutine.